include/linux/medusa/l4/interface.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _L4_INTERFACE_H
+#define _L4_INTERFACE_H
+
+typedef enum {
+	AUTHS_NOT_REACHED = -2,
+	AUTHS_ERR = -1,
+	AUTHS_FORCE_ALLOW = 0,
+	AUTHS_DENY = 1,
+	AUTHS_SKIP = 2,
+	AUTHS_ALLOW = 3,
+	AUTHS_DBG_ALLOW = 10
```

Comment:
AUTHS_DBG_ALLOW can be 4; is it any reason for value 10?

Reply:
The reason behind is that this is not coming from authorization server. If we will implement new answer, auth. server do not know that we used 4 for another code and logically will just increase the highest code value by one to represent new answer and this can result in a serious bug if we forget about it. Thats why I selected 10 as a starting point for codes for development/ debug purposes

Reply:
so, in that case the value should be negative... because this approach is not 100% safe